docs/EN_US/HPCCClientTools/CT_Mods/CT_ESDL_CLI.xml

```diff
@@ -1049,6 +1049,29 @@
           appropriate gateway's connection information, then use that
           information to create the actual gateway request connection.</para>
         </sect3>
+
+        <sect3>
+          <title>Configuring ESDL binding for Proxy Mode methods</title>
+
+          <para>You can specify a proxy for ESDL service methods on another
+          ESP instance. Set up the proxy in the dynamic configuration
+          associated with the dESDL service. </para>
+
+          <para>Under the Methods tag where you would add Method tags, you can
+          also add Proxy tags, as shown here:</para>
+
+          <programlisting>&lt;Methods&gt; 
+  &lt;Method name="myMethod" url="http://10.45.22.1:292/somepath"/&gt;
+  &lt;Method name="myMethod2" url="http://10.45.22.1:292/somepath"/&gt; 
+  &lt;Proxy method="myMethod3" address="http://10.45.22.1:292" /&gt; 
```

Comment:
"address" attribute was correct, but is changing to "forwardTo".

Reply:
@JamesDeFabia Looks like address has been changed to forwardTo below, but the one example above hasn't been changed.